src/pages/ResearcherReview.js

```diff
@@ -85,8 +85,11 @@ class ResearcherReview extends Component {
     } else if (voteStatus === "false") {
       status = "rejected";
     }
-    let userStatus = { status: status, rationale: rationale, roleId: 5 };
-    User.registerStatus(userStatus, this.props.match.params.dacUserId).then(
+    let updatedUser = this.state.user;
+    updatedUser.status = status;
+    updatedUser.rationale = rationale;
```

Comment:
The rationale isn't updating on submission, but I can see the rationale and status on the request payload. I suspect the UI is doing it's part but the back-end isn't, so the lack of functionality is not an issue here.

Reply:
yeah I noticed that too do you think I should file a ticket?

Reply:
Yeah, without it admins can't vote to approve or disapprove of new users.

Reply:
It may be a good idea to hold off on merging this for now, current develop has the rationale and vote updating correctly (because it's using the old endpoint). Once the rationale fix is in for the new update this PR should function correctly.